specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/security.json

```diff
@@ -920,6 +920,116 @@
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/InformationProtectionPolicy"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/CloudError"
+                        }
+                    }
+                }
+            },
+            "put": {
+                "x-ms-examples": {
+                    "Create or update an information protection policy for a management group": {
+                        "$ref": "./examples/InformationProtectionPolicies/CreateOrUpdateInformationProtectionPolicy_example.json"
+                    }
+                },
+                "tags": ["InformationProtectionPolicies"],
+                "description": "Details of the information protection policy.",
+                "operationId": "InformationProtectionPolicies_CreateOrUpdate",
+                "parameters": [{
+                        "$ref": "#/parameters/ApiVersion"
+                    }, {
+                        "$ref": "#/parameters/Scope"
+                    }, {
+                        "$ref": "#/parameters/InformationProtectionPolicyName"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/InformationProtectionPolicy"
+                        }
+                    },
+                    "201": {
+                        "description": "Created",
+                        "schema": {
+                            "$ref": "#/definitions/InformationProtectionPolicy"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/CloudError"
+                        }
+                    }
+                }
+            }
+        },
+        "/{scope}/providers/Microsoft.Security/informationProtectionPolicies": {
+            "get": {
+                "x-ms-examples": {
+                    "Get information protection policies": {
+                        "$ref": "./examples/InformationProtectionPolicies/ListInformationProtectionPolicies_example.json"
+                    }
+                },
+                "tags": ["InformationProtectionPolicies"],
+                "description": "Information protection policies of a specific management group.",
+                "operationId": "InformationProtection_List",

```

Comment:
>InformationProtection_List [](start = 32, length = 26)

should be `InformationProtectionPolicies_List`

Reply:
Fixed. Thanks